### DISCUSSION TITLE
cp (copy) using mc (minio) utility is resulting with “ No such file or directory” error.

### DISCUSSION THREAD

**Discussion by KalyanLearning** [minio/minio#16847] [Q&A] [answered]
cp (copy) using mc (minio) utility is resulting with “ No such file or directory” error. 
As part of CICD, I am trying to copy  the .jar file to Minio bucket with mc cp command from bastion host.

**Comment by klauspost** [comment, marked as answer]
Please post exactly what you are doing and which files are placed where.

**Reply by KalyanLearning** [reply]
Hello, 
I tried to Copy .jar files from dev bastion server to Minio.

Commad used: 
$ **mc cp test.jar  https://minio.*<url>**

Error : no such file or directory. We are sure that file exists and we dought the Cp cmd is not working.

**Reply by klauspost** [reply]
You are using incorrect syntax. 

The documentation is here: https://min.io/docs/minio/linux/reference/minio-mc.html
